11-http-rest/lab/README.md

```diff
@@ -0,0 +1,127 @@
+# HTTP, REST, JSON и HttpClient
```

Comment:
Why this file is part of this pull request?

Reply:
Misplaced, removed it.